src/main/resources/api.yaml

```diff
@@ -147,6 +147,41 @@ paths:
           description: Sprint created succesfully
           schema:
             $ref: '#/definitions/SprintDto'
+  /projects/{projectId}/sprints/{sprintId}:
+    patch:
+      tags:
+        - projects
+        - sprints
+      summary: Requests a change to an existing sprint
+      operationId: patchSprint
```

Comment:
Nie wiem, jak jest ładniej - to `patchSprint` w sumie mało mówi, a umożliwiamy tylko bardzo konkretną operację tutaj - zamknięcie sprintu z podaniem tych dwóch czasów, jeśli sprint jest w odpowiednim stanie - `closeSprint` explicite nie byłoby ładniejsze?

Reply:
`modifySprint`? Nie wiadomo co w przyszłości będzie robić ta metoda.

Reply:
Może być